Makefile.core.mk

```diff
@@ -168,7 +171,7 @@ run: gen ## Run a controller from your host.
 # docker build -t ${IMAGE} --build-arg GIT_TAG=${GIT_TAG} --build-arg GIT_REVISION=${GIT_REVISION} --build-arg GIT_STATUS=${GIT_STATUS} .
 .PHONY: docker-build
 docker-build: build ## Build docker image with the manager.
```

Comment:
Not sure it would work because the `build` target uses `build-amd64` (cf. [here](https://github.com/maistra/istio-operator/blob/maistra-3.0/Makefile.core.mk#L162)). That means the arm64 container image would use the wrong arch of the manager binary.

Reply:
I made the change only for the target docker-build and I tested on my fedora VM with amd64 and works. The operator was deployed successfully on arm OCP cluster

Reply:
This is because I set --platform=linux/arm64 flag and value for the docker build command